hat/app/org/hatdex/hat/api/controllers/RichData.scala

```diff
@@ -209,6 +208,19 @@ class RichData @Inject() (
     } getOrElse Set()
 
   // Extract to namespace library
+  private def authorizeEndpointDataRead(
+      namespace: String,
+      appStatus: Option[HatApplication]
+    )(implicit
+      user: HatUser,
+      authenticator: HatApiAuthEnvironment#A): Boolean = {
+
+    val role = NamespaceWrite(namespace)
```

Comment:
`authorizeEndpointDataRead` checks for `NamespaceWrite` ?

Reply:
This needs to be confirmed with @whiteshadow-gr , updating a record needs ```NamespaceRead``` https://github.com/dataswift/HAT2.0/blob/main/hat/app/org/hatdex/hat/api/controllers/RichData.scala#L221 so this is currently a pure guess as the permission system is not documented anywhere

Reply:
confirmation from @whiteshadow-gr on ```NamespaceRead``` updated accordingly